release-notes/6.0/api-diff/.Net/6.0.0_System.md

```diff
@@ -0,0 +1,336 @@
+# System
+
+``` diff
+ namespace System {
+     public class ArgumentNullException : ArgumentException {
++        public static void ThrowIfNull(object argument, string paramName = null);
```

Comment:
This doesn't list the `[CallerArgumentExpression]` attribute on `paramName`. Should it?